src/index.ts

```diff
@@ -427,82 +630,150 @@ export default class OneSignal {
         RNOneSignal.addTriggers(triggers);
     }
 
-    static removeTriggersForKeys(keys) {
+    /**
+     * Removes a list of triggers based on a collection of keys.
+     * @param  {string[]} keys
+     * @returns void
+     */
+    static removeTriggersForKeys(keys: string[]): void {
         if (!isObjectNonNull(RNOneSignal)) return;
 
         RNOneSignal.removeTriggersForKeys(keys);
     }
 
-    static removeTriggerForKey(key) {
+    /**
+     * Removes a list of triggers based on a key.
+     * @param  {string} key
+     * @returns void
+     */
+    static removeTriggerForKey(key: string): void {
         if (!isObjectNonNull(RNOneSignal)) return;
         RNOneSignal.removeTriggerForKey(key);
     }
 
-    static getTriggerValueForKey(key) {
+    /**
+     * Gets a trigger value for a provided trigger key.
+     * @param  {string} key
+     * @returns Promise<string>
```

Comment:
make sure returns on all comments match the actual return values

Reply:
Done